docs/building-images.md

```diff
@@ -75,6 +75,7 @@ operatingSystem:
   kernelArgs:
   - arg1
   - arg2
+  enableFips: true
```

Comment:
I'm wondering if we should use capitalise the param e.g. `enableFIPS` - WDYT?

Reply:
I'd prefer to have it capitalised, but I was just going with other examples :)

Reply:
My comment is pretty much based on the fact that it's an acronym.

Reply:
It's tricky. We have

```
httpProxy
sshKeys
```

which are technically acronyms, but we don't capitalize. But we also have:

```
noGPGCheck
```

... should we roll the EIB Decision Dice?

Reply:
We spoke about it in chat and let's go with `enableFederalInformationProcessingStandard` to make it simpler